bootstrap.css

```diff
@@ -15,7 +15,6 @@ body {
 span{
   border:2px;
 }
-article,
```

Comment:
y this? article s needed
![picture](https://cloud.githubusercontent.com/assets/4362065/16818459/8bd7006a-4965-11e6-959d-43c1bdca6278.jpg)